src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -718,6 +722,27 @@ private void InitializePath(PathSetting path)
                 Directory.CreateDirectory(path.FullPath);
         }
 
+        private static void ValidateAnalyzers(IndexDefinition definition)
+        {
+            if (definition.Fields == null)
+                return;
+
+            foreach (var kvp in definition.Fields)
+            {
+                if (string.IsNullOrWhiteSpace(kvp.Value.Analyzer))
+                    continue;
+
+                try
+                {
+                    IndexingExtensions.GetAnalyzerType(kvp.Key, kvp.Value.Analyzer);
+                }
+                catch (Exception e)
+                {
+                    throw new IndexCompilationException(e.Message, e);
```

Comment:
The message can be hard to figure out. Better put it : "Failed to load analyze ANALYZER NAME", and then the exception with the details